### DISCUSSION TITLE
How can I override/customize --version for my .NET 4.8 project?

### DISCUSSION THREAD

**Discussion by Xorcist77** [dotnet/command-line-api#2449] [Q&A]
Presently, --version returns Major.Minor.Patch.Build, and even changing it to Major.Minor.Patch in the AssemblyInfo.cs continues to return a 0 for .Build. I'd like to only return a semantically formatted version with only Major.Minor.Patch being displayed. Is there a method I can override for this?

**Comment by KalleOlaviNiemitalo** [comment]
See <https://github.com/dotnet/command-line-api/issues/2009>.

**Reply by Xorcist77** [reply]
Thank you, but it appears that System.CommandLine 2.0.0-beta4.23178.3 is not generally available on Nuget yet (the latest is 2.0.0-beta4.22272.1). Not sure if that matters, starting to dig into the source to see what I can find, but presently VersionOption seems to be inaccessible due to its protection level inside my main.

`static async Task<Int32> Main(String[] args) {}`

**Reply by Balkoth** [reply]
You nee to add this nuget source: https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json
